google-oauth-client/src/main/java/com/google/api/client/auth/openidconnect/IdTokenVerifier.java

```diff
@@ -280,5 +402,163 @@ public Builder setAcceptableTimeSkewSeconds(long acceptableTimeSkewSeconds) {
       this.acceptableTimeSkewSeconds = acceptableTimeSkewSeconds;
       return this;
     }
+
+    /** Returns an instance of the {@link Environment} */
+    public final Environment getEnvironment() {
+      return environment;
+    }
+
+    /** Sets the environment. Used mostly for testing */
+    public Builder setEnvironment(Environment environment) {
```

Comment:
Would you check whether this needs to be public?

Reply:
These ones should be package private - `Environment` interface is already package private.

Reply:
Yes, its a test infra, thanks!